### PR TITLE
TAJO-1745: Add positive and negative test methods.

### PR DESCRIPTION
**Pull request by hyunsik**
[Review on Reviewable](https://reviewable.io/reviews/apache/tajo/681)

### CODE REVIEW

**Comment by jinossy** [comment]
Trigger travis please?

File: tajo-core/src/test/java/org/apache/tajo/QueryTestCaseBase.java [hunk 1]
```diff
@@ -18,6 +18,9 @@
 
 package org.apache.tajo;
 
+import com.facebook.presto.hive.shaded.com.google.common.base.Function;
+import com.facebook.presto.hive.shaded.com.google.common.collect.Collections2;
```
Comment (jinossy) [inline tajo-core/src/test/java/org/apache/tajo/QueryTestCaseBase.java]: Could you change to google collection?

Comment (hyunsik) [inline tajo-core/src/test/java/org/apache/tajo/QueryTestCaseBase.java]: fixed

**Comment by hyunsik** [comment]
rebased

**Comment by hyunsik** [comment]
rebased

**Comment by jinossy** [comment]
Would you check failure?
```
  TestSelectQuery.testPositives:59->QueryTestCaseBase.runNegativeTests:508->QueryTestCaseBase.getNegativeQueryFiles:852 » IO
  TestSelectQuery.testNegatives:64->QueryTestCaseBase.runNegativeTests:508->QueryTestCaseBase.getNegativeQueryFiles:852 » I

Tests run: 69, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 68.622 sec <<< FAILURE! - in org.apache.tajo.engine.query.TestSelectQuery
testPositives(org.apache.tajo.engine.query.TestSelectQuery)  Time elapsed: 0.005 sec  <<< ERROR!
java.io.IOException: Cannot find file:/Users/jhkim/tajo/asf/tajo/tajo-core-tests/target/test-classes/queries/TestSelectQuery/nagative
        at org.apache.tajo.QueryTestCaseBase.getNegativeQueryFiles(QueryTestCaseBase.java:852)
        at org.apache.tajo.QueryTestCaseBase.runNegativeTests(QueryTestCaseBase.java:508)
        at org.apache.tajo.engine.query.TestSelectQuery.testPositives(TestSelectQuery.java:59)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
```

**Comment by hyunsik** [comment]
rebased and fixed.

**Comment by jinossy** [comment]
+1 LGTM

**Pull request closed without merging**